drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt

```diff
@@ -168,6 +168,18 @@ class DropInViewModel(
         }
     }
 
+    fun onDropInServiceConnected() {
+        session?.let { session ->
```

Comment:
Can we add a log in case the session is null? I'm afraid we might miss out on any potential bugs if this method is called at the wrong time. A bit too late now but maybe we can do this in another PR.